dvc/remote/__init__.py

```diff
@@ -1,13 +1,10 @@
 from ..tree import get_cloud_tree
 from .base import Remote
 from .local import LocalRemote
-from .ssh import SSHRemote
 
 
 def get_remote(repo, **kwargs):
     tree = get_cloud_tree(repo, **kwargs)
     if tree.scheme == "local":
         return LocalRemote(tree)
-    if tree.scheme == "ssh":
-        return SSHRemote(tree)
```

Comment:
We never needed an SSHRemote, as all of the custom stuff was actually about SSHCache.